pyart/graph/radardisplay.py

```diff
@@ -586,7 +586,7 @@ def plot_vpt(self, field, mask_tuple=None, vmin=None, vmax=None,
         # set up the time axis
         if time_axis_flag:
             self._set_vpt_time_axis(ax, date_time_form=date_time_form, tz=tz)
-            x = datetimes_from_radar(radar)
+            x = datetimes_from_radar(self._radar)
```

Comment:
Caught this too, a unit test to test this may be good to add, I wrote one in #457 if you want to take that one.